docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -123,6 +125,74 @@ mc mb --ignore-existing \
   "${CLUSTER_ID}/${CLUSTER_ID}-bootstrap-ignition"
 ----
 
+=== Upload Red Hat CoreOS image
+
+. Export the Athorization header for the Cloudscale API.
+
+[source,console]
+----
+export AUTH_HEADER="Authorization: Bearer ${CLOUDSCALE_TOKEN}"
+----
+
+[NOTE]
+====
+The variable `CLOUDSCALE_TOKEN` could be used directly.
+Exporting the variable `AUTH_HEADER` is done to be compatible to the https://www.cloudscale.ch/en/api/[Cloudscale API documentation].
+====
+
+. Check if image already exists
+
+[source,console]
+----
+curl -sH "$AUTH_HEADER" https://api.cloudscale.ch/v1/custom-images | jq -r '.[] | select(.slug == "rhcos-4.7") | .href'
+----
+
+[NOTE]
+====
+If a URL is printed to the output, you can skip the next steps and directly jump to the next section.
```

Comment:
```suggestion
If an URL is printed to the output, you can skip the next steps and directly jump to the next section.
```

Reply:
No, Microsoft thinks otherwise. I had it that way but then the vale check failed.